src/lib/htmlInputPropsUtils.js

```diff
@@ -44,7 +44,10 @@ export const partitionHTMLInputProps = (props, htmlProps = htmlInputProps) => {
   const inputProps = {}
   const rest = {}
 
-  _.forEach(props, (val, prop) => _.includes(htmlProps, prop) ? (inputProps[prop] = val) : (rest[prop] = val))
```

Comment:
Statement `_.includes(htmlProps, prop) ? (inputProps[prop] = val) : (rest[prop] = val)` will return prop's value, when value will be `false` the `forEach` will stop execution.